job/job.go

```diff
@@ -101,7 +100,7 @@ type Job struct {
 	JobType jobType `json:"type"`
 
 	// Custom properties for the remote job type
-	RemoteProperties RemoteProperties `json:"remote_properties"`
+	RemoteProperties
```

Comment:
due to our Post.CreateJOB request on FE has schema
```
{"name":"segment-sync-01","type":1,"owner":"paul@primedata.ai","schedule":"R1/2020-06-28T19:30:16.828696-07:00/PT10S","epsilon":"PT5S","retries":0,"url":"https://graph.facebook.com/v7.0/act_ACCOUNT_ID/customaudiences","method":"POST","body":"{\"name\":\"My new Custom Audience\"',\"subtype\":\"CUSTOM\",\"description\":\"People who purchased on my website\"',\"customer_file_source\":\"USER_PROVIDED_ONLY\",\"access_token:\"\"}","timeout":0,"expected_response_codes":[]}
```
we need Struct composition like this

Reply:
I've tested FE feature and found this bug

Reply:
Couple points:
1. Sorry, what is FE?  (Frontend?  You mean the dashboard, or the JSON API, or your own code?)
2. Unfortunately I don't want to change the structure of `job.Job` because that will break compatibility with active users who are using the existing structure to store & retrieve jobs in the database.  It's not clear to me what is wrong but I think whatever is talking to Kala needs to adjust itself rather than the structure of the data within Kala.

Reply:
```
{
  "name": "segment-sync-01",
  "type": 1,
  "owner": "paul@primedata.ai",
  "schedule": "R1/2020-06-28T19:30:16.828696-07:00/PT10S",
  "epsilon": "PT5S",
  "retries": 0,
  "url": "https://graph.facebook.com/v7.0/act_ACCOUNT_ID/customaudiences",
  "method": "POST",
  "body": "{\"name\":\"My new Custom Audience\"',\"subtype\":\"CUSTOM\",\"description\":\"People who purchased on my website\"',\"customer_file_source\":\"USER_PROVIDED_ONLY\",\"access_token:\"\"}",
  "timeout": 0,
  "expected_response_codes": []
}
```
Above json is current payload from Dashboard. It's 400 error on creating Job. because URL is not bind into Job.
RemoteProperties must be composited in Job

Reply:
```
ERRO[1113] Invalid Remote Job. Job's must contain a Name and a url field 
ERRO[1113] Error occurred when initializing the job: Invalid Remote Job. Job's must contain a Name and a url field 
```